code/modules/mob/living/carbon/xenomorph/facehuggers.dm

```diff
@@ -567,9 +573,21 @@
 		user.Unconscious(2 SECONDS)
 	attached = TRUE
 	go_idle(FALSE, TRUE)
-	addtimer(CALLBACK(src, PROC_REF(Impregnate), user), IMPREGNATION_TIME)
+	timerid = addtimer(CALLBACK(src, PROC_REF(Impregnate), user), IMPREGNATION_TIME, TIMER_STOPPABLE)
+	RegisterSignal(src, COMSIG_TRY_STRIP, PROC_REF(on_strip))
+
+/obj/item/clothing/mask/facehugger/proc/on_strip(I, M)
+	SIGNAL_HANDLER
+	if(timeleft(timerid) <= HUMAN_STRIP_DELAY)
+		deltimer(timerid)
+		timerid = addtimer(CALLBACK(src, PROC_REF(Impregnate), M), HUMAN_STRIP_DELAY + 1, TIMER_STOPPABLE)
```

Comment:
this whole thing seems pretty lame, in that its physically impossible to get someone impregnated as long as someone can literally just click them once. Like, the threshold is even lower with this pr than removal by flare, at least with that method you need an item.

Reply:
Flares are a set and forget item that you can just pick up off the ground and then throw at someone, not to mention that you just need to get them on fire and it's now instantly off of them. At least with this you can be shuffled and delayed and it takes time to remove it, it also means a carrier could shuffle a marine to ensure the pregnancy. I did initially want to make it so you could only reset it once but i decided against it as i thought that wouldn't be wanted by the maintainers.

Reply:
you don't need to open the strip menu, its just m1 on help intent, but oh well

Reply:
> you don't need to open the strip menu, its just m1 on help intent, but oh well

I know it is, still takes the 4 second interaction to do though and it gives you exactly 4 seconds, lets say you are 3 seconds in, get shuffled away from them, that means you only have 1 second to click them again or they will be impregnated. Would you prefer if i made it so that you only get the timer reset once though?